style.css

```diff
@@ -769,30 +1020,53 @@ span {
     border-radius: 7% 0 0 0;
   }
 
-  #my_skills p {
-    align-self: center;
-  }
-
   .last-sect {
     background-color: #fff;
     border-radius: 0;
   }
 
-  .skills {
-    display: flex;
-    align-items: baseline;
-    justify-content: flex-start;
-    padding-left: 0;
-    margin-left: 0;
-  }
-
-  footer .skills { margin-left: 0; }
-
   .contact input {
     border-radius: 0;
   }
 
   #get-resume {
     padding: 8px 6px;
   }
+
+  .pop-header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .pop-window {
+    height: 100vh;
+    width: 80%;
+    margin-left: 10%;
+  }
```

Comment:
- [ ] You have done great to ensure that When the user clicks (or taps) the button to check project details, the popup with details about the project appears, and when the user clicks (or taps) the close (X) button, the popup disappears 👏 👍. However, we should try as much as possible to ensure that we stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=Z3hCN1aRrx7pYjUK-0). This implies to apply the blur effect as seen in the Figma. In addition, you need to ensure that there is the spacing between the popup and the portfolio borders in all sides of the popup window to match the Figma design ☺️.
 
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/225569722-182a18ec-e6c8-405a-87f5-1198919a4130.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/225569850-908d864a-7b14-4bf6-8136-d5d9bd53b456.png)